src/client/components/author-credit-display.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 2020  Sean Burke
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import {map as _map} from 'lodash';
+
+
+function AuthorCreditDisplay({names}) {
+	const nameElements = _map(names, (name) => (
+		<span key={`author-credit-${name.authorCreditID}-${name.position}`}>
+			<a href={`/author/${name.authorBBID}`}>
+				{name.name}
+			</a>
+			{name.joinPhrase}
+		</span>
+	));
+
+	return (
+		<span>
+			{nameElements}
```

Comment:
What do you think of `by {nameElements}`?

Reply:
This actually reminds me of something I'd like to have longer-term; I'd like to be able to have lead-in text for an author credit. There are a lot of cases out there where the credit is something like "Edited by Foo", and it'd be nice to be able to represent that in data. With the potential for lead-in text, the "by" would be out-of-place, but the lead-in text itself bears further discussion.

Reply:
Looking at the `author_credit` table in the database, there is a `begin_phrase` text column that seems to be there for that.
It's not part of the sql schema though; I'm not sure when it was added but it was removed here: https://github.com/bookbrainz/bookbrainz-site/commit/7cc6f6fdad569e5595ba622321b4a7e5535b1f19
Not sure why, it seems like it would be useful.